internal/runtime/hcsv2/sandbox_container.go

```diff
@@ -97,6 +97,9 @@ func setupSandboxContainerSpec(ctx context.Context, id string, spec *oci.Spec) (
 	// also has a concept of a sandbox/shm file when the IPC NamespaceMode !=
 	// NODE.
 
+	// Force the parent cgroup into our /containers root
+	spec.Linux.CgroupsPath = "/containers/" + id
```

Comment:
@katiewasnothere - For the cgroup stats work you are doing with this change you can now predict the path very easily for each subsystem. The root is "/containers/{id}" under each root. So:
```
/sys/fs/cgroup/cpu/containers/{id}
/sys/fs/cgroup/memory/containers/{id}
etc.
```